plugins/woocommerce-admin/client/wp-admin-scripts/product-tracking/shared.ts

```diff
@@ -29,7 +62,66 @@ const getProductData = () => {
 		manage_stock: (
 			document.querySelector( '#_manage_stock' ) as HTMLInputElement
 		 )?.value,
```

Comment:
The values recorded for:

* `is_downloadable`
* `is_virtual`
* `manage_stock`

are all incorrect.

They always record as `on`, since that is the `value` of the corresponding input field.

Instead, we should be checking the `.checked` property.

We should correct this since we are right in this code now, and let @pmcpinto know that any analysis of the following Tracks events using those properties prior to this fix are invalid:

* `wcadmin_product_copy`
* `wcadmin_product_delete`
* `wcadmin_product_publish_widget_edit`
* `wcadmin_product_publish_widget_save`

Reply:
@mattsherman thanks for the heads up. I agree that the issues should be fixed asap. I'm assuming this is affecting wcadmin_product_update and wcadmin_product_add_publish, correct?

Reply:
@pmcpinto No, I believe this only affects the Tracks events I listed.